data_model.md

```diff
@@ -250,13 +243,13 @@ of forecast parameters):
 ### Aggregates
 {: .anchor}
 
-Each Aggregate must be associated with one or more Observations. Solar
-Forecast Arbiter will calculate aggregated observation data generated
-from these Observations. Each aggregate will also provide a list of
+Each Aggregate must be associated with one or more Observations. The Solar
+Forecast Arbiter calculates aggregated observation data
+from these Observations. Each aggregate also provides a list of
```

Comment:
@alorenzo175 are these sentences about aggregates and sites correct? Seems like the implementation does not actually do this.

Reply:
Not correct. Right now only provides list of observations from which you can get the sites.